tests/util/util.go

```diff
@@ -0,0 +1,229 @@
+package util
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/AthenZ/athenz/libs/go/athenz-common/log"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/portforward"
+	"k8s.io/client-go/transport/spdy"
+)
+
+var k8sClient *kubernetes.Clientset
+var k8sConfig *rest.Config
+var stopChan chan struct{}
+
+func init() {
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Printf("error getting user home dir: %v\n", err)
+		os.Exit(1)
+	}
+	kubeConfigPath := filepath.Join(userHomeDir, ".kube", "config")
+	fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+	k8sConfig, err = clientcmd.BuildConfigFromFlags("", kubeConfigPath)
+	if err != nil {
+		println("config build error")
+	}
+
+	k8sClient, err = kubernetes.NewForConfig(k8sConfig)
+	if err != nil {
+		println("config build error")
+	}
+}
+
+// Kubectl execute
+func Kubectl(cmdstr string) (string, error) {
+	cmdf := strings.Fields(cmdstr)
+	cmd := exec.Command("kubectl", cmdf...)
+	sout, err := cmd.Output()
+	return string(sout), err
+}
```

Comment:
Not related to this PR but it would be better if we do not depend on binaries installed in the VM when we are already using k8s client-go. 

This would require changes to the actual tests too.